src/main/java/core/basesyntax/validation/OperationValidationImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.validation;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+
+public class OperationValidationImpl implements OperationValidation {
+    private static final int CORRECT_LENGTH_OF_SPLITTED_ROW = 3;
+    private static final int INDEX_FRUIT_AMOUNT = 2;
+    private static final int INDEX_OPERATION_TYPE = 0;
+    private static final int INDEX_FRUIT_TYPE = 1;
+
+    private FruitDao fruitDao = new FruitDaoImpl();

```

Comment:
you should not depend storage when performing validation - let's remove this field and the check